### PR TITLE
Add direct emission values to config

### PR DESCRIPTION
**Pull request by gwpicard**
Fixes ELE-947
Fixes ELE-954

- adding direct emission factors per production mode and sources
- discharge and unknown values TBD (need to be calculated)

### CODE REVIEW

**Review by pierresegonne — CHANGES_REQUESTED** [review]
I suggest we rename the files to remove the `_global` tag, as we currently don't know how the solution for regional emission factors will look like

**Comment by pierresegonne** [comment]
Nice let's include unknown and we'll be good 👏

**Comment by pierresegonne** [comment]
Also fixes ELE-954

File: config/co2eq_parameters_direct.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        "source": "EIA 2020/BEIS 2021",
+        "value": 406
+      },
+      "solar": {
+        "source": "IPCC 2014",
+        "value": 0
+      },
+      "unknown": {
+        "_comment": "assume conventional",
+        "source": "assumes thermal (coal, gas, oil or biomass)",
```
Comment (gwpicard) [inline config/co2eq_parameters_direct.json:59]: @pierresegonne do you agree with the phrasing here (and in the _comment)?

Comment (pierresegonne) [reply config/co2eq_parameters_direct.json:59]: Let's just remove biomass

**Review by pierresegonne — APPROVED** [review]
Nice one

**Pull request merged**